crates/router/src/connector/aci/transformers.rs

```diff
@@ -91,12 +91,148 @@ impl TryFrom<&api_models::payments::WalletData> for PaymentDetails {
                     account_id: None,
                 }))
             }
-            _ => Err(errors::ConnectorError::NotImplemented(
+            api_models::payments::WalletData::MbWayRedirect( _ ) |
+            api_models::payments::WalletData::AliPayRedirect( _ ) => {
```

Comment:
Please do the same for the rest of the match arms. Please use OR.

Reply:
Hey @tanbirali ,
Instead of writing the same error message for each case, use `OR` between each case, like this:
`  api_models::payments::WalletData::MbWayRedirect( _ ) |
            api_models::payments::WalletData::AliPayRedirect( _ ) => {`
            
   follow the same pattern for all the arms.

Reply:
Okay I have made the changes could you please check and let me know if I understood you correctly.

Reply:
Hey @tanbirali , you got it right.

Pls address the CI fails

Run `cargo +nightly fmt` for formatting.